demos/browser/package.json

```diff
@@ -32,8 +32,8 @@
   },
   "dependencies": {
     "amazon-chime-sdk-js": "file:../..",
-    "aws-sdk": "^2.751.0",
-    "bootstrap": "4.5.0",
+    "aws-sdk": "^2.755.0",
+    "bootstrap": "4.5.1",
```

Comment:
We have removed `^` to use `4.5.0`. Please use `^` to update to all minior/patch versions, e.g. any version < 5.0.0.

Reply:
sure, will do